main.py

```
@@ -45,5 +34,12 @@ def process_box_score(event, context):
         pc = pcweb.PcWeb('256')  # '1000' for testing
         # pc.send_to_thromer('stuff happened', '\n'.join(messages))
         for message in messages:
-            pc.league_chat('%s [Day %s]' % (message, blob.metadata['day']), trailing_whitespace=int(blob.metadata['year']) % 5)
+            metadata = cast(dict[str, Any], blob.metadata)
+            pc.league_chat('%s [Day %s]' % (message, metadata['day']), trailing_whitespace=int(metadata['year']) % 5)
     
+@app.route('/', methods=['POST'])
+def process_box_score_eventarc():
```

Ruff (0.11.9)

Missing return type annotation for public function `process_box_score_eventarc`

Add return type annotation: `str`

(ANN201)